tests/app/views/contexts/test_submitted_response_context.py

```diff
@@ -140,6 +150,21 @@ def fake_questionnaire_store():
     return questionnaire_store
 
 
+def fake_questionnaire_store_social():
```

Comment:
We could pass in the required metadata to the existing questionnaire_store rather than creating a new method here?

Reply:
Ok refactored all of it now so there is only one `fake_questionnaire_store`.